cinema_service/settings.py

```diff
@@ -140,3 +141,40 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_PERMISSION_CLASSES": [
+        "cinema.permissions.IsAdminOrIfAuthenticatedReadOnly",
+    ],
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework_simplejwt.authentication.JWTAuthentication",
```

Comment:
you set here default auth class, but in your views you still use TokenAuthentication, thus you need to remove it

Reply:
@Yuruch Did you forget about this ?)

Reply:
@DevSpaciX but i had removed TokenAuthentication from all the views in the last commit